pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -359,7 +359,7 @@ func (am *Alertmanager) buildReceiverIntegrations(receiver *alertingNotify.APIRe
 		return nil, err
 	}
 	s := &sender{am.NotificationService}
-	img := newImageStore(am.Store)
```

Comment:
There are a lot of renamings like `imageStore -> imageProvider` in this PR, which make the diff biggern